.gitignore

```
@@ -0,0 +1,4 @@
+.DS_Store
+.idea/
+.pre-commit-config.yaml
+coverage.out
```

Let's add the trailing line.

```suggestion
coverage.out

```